src/main/java/com/softserve/model/SubCategory.java

```diff
@@ -0,0 +1,43 @@
+package com.softserve.model;
+
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class SubCategory {
+
+	@Id
+	@Column(name = "id_category")
```

Comment:
is any of those IDs autoincremental? if so, please specify with appropriate annotation.
confirm with Joceline